include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1991,16 +1991,84 @@ class FEEvaluation : public FEEvaluationAccess<dim,n_components_,Number>
                  const bool evaluate_gradients,
                  const bool evaluate_hessians = false);
 
+  /**
+   * Evaluates the function values, the gradients, and the Laplacians of the
+   * FE function given at the DoF values in the input array @p values_array at
+   * the quadrature points on the unit cell. If multiple components are
+   * involved in the current FEEvaluation object, the sorting in values_array
+   * is such that all degrees of freedom for the first component come first,
+   * then all degrees of freedom for the second, and so on. The function
+   * arguments specify which parts shall actually be computed. Needs to be
+   * called before the functions @p get_value(), @p get_gradient() or @p
```

Comment:
> Needs to be called before... give useful information

I would reword it, maybe

This function has to be called first so that... give useful info

@jppelteret can the original be improved or am I just confused?